From d62c9e281ed6bf505007dcfe87b3017d381d68df Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 10:02:04 +0800
Subject: [PATCH] Test case4 for Delete an undefault value FeatureFlag PCD

Description: Delete the FeatureFlag PCD with the different value ("gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|TRUE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..e97784c755 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -176,7 +176,7 @@
 #
 ################################################################################
 [PcdsFeatureFlag]
-gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|TRUE
+#gUefiOvmfPkgTokenSpaceGuid.PcdSmmSmramRequire|TRUE
 gEfiMdeModulePkgTokenSpaceGuid.PcdPeiFullPcdDatabaseEnable|TRUE
 
 
-- 
2.27.0.windows.1

